src/api/todos.ts

```diff
@@ -6,3 +6,25 @@ export const getTodos = (userId: number) => {
 };
 
 // Add more methods here
+type Data = {
+  title?: string,
+  userId?: number,
+  completed: boolean,
+};
+
+type ObjData = {
+  id: number,
+  userId: number,
+  completed: boolean,
+  title: string,
+  createdAt: string,
+  updatedAt: string,
+};
+
+export const postTodos = (userId: number, data: Data): Promise<ObjData> => {
```

Comment:
```suggestion
export const postTodo = (userId: number, data: Data): Promise<ObjData> => {
```
You are posting single Todo, aren't you?